gp-includes/things/translation-set.php

```diff
@@ -254,11 +254,13 @@ public function import( $translations, $desired_status = 'current' ) {
 			/**
 			 * Filter the the status of imported translations of a translation set.
 			 *
-			 * @since 1.0.0
+			 * @since 2.2.0
 			 *
 			 * @param string $status The status of imported translations.
+			 * @param Translation_Entry $entry  Translation entry object to import.
+			 * @param GP_Translation|false $translated The previous translation.
 			 */
-			$entry->status = apply_filters( 'gp_translation_set_import_status', $is_fuzzy ? 'fuzzy' : $desired_status );
+			$entry->status = apply_filters( 'gp_translation_set_import_status', $is_fuzzy ? 'fuzzy' : $desired_status, $entry, false );
```

Comment:
The argument $entry (see below) was added in 0448424 but only in one of the occurrences of the filter.